client/signup/jetpack-connect/authorize-form.jsx

```diff
@@ -106,23 +122,16 @@ const LoggedInForm = React.createClass( {
 		return this.props.jetpackConnectAuthorize && this.props.jetpackConnectAuthorize.isAuthorizing;
 	},
 
-	dismissNotice() {
-		debug( 'Notice dissmissed' );
-	},
-
 	renderNotices() {
-		const { authorizeSuccess, authorizeError } = this.props.jetpackConnectAuthorize;
-		if ( authorizeSuccess ) {
-			return <JetpackConnectNotices noticeType="authorizeSuccess" onDismissClick={ this.dismissNotice } />;
-		}
+		const { authorizeError } = this.props.jetpackConnectAuthorize;
 		if ( authorizeError ) {
-			return <JetpackConnectNotices noticeType="authorizeError" onDismissClick={ this.dismissNotice } />;
+			return <JetpackConnectNotices noticeType="authorizeError" />;
 		}
 		return null;
 	},
 
 	renderFormControls() {
-		const { queryObject, isAuthorizing, autoAuthorize, authorizeSuccess } = this.props.jetpackConnectAuthorize;
+		const { queryObject, isAuthorizing, autoAuthorize, authorizeSuccess, plansURL, siteReceived } = this.props.jetpackConnectAuthorize;
```

Comment:
The PlansURL is calculated on the API after the authorization is complete. It refers to `wordpress.com/plans/$site` - and since we don't know what `$site` is until the authorization is complete and we have a real `siteSlug`, it will have to be calculated on the API. Perhaps the API can simple calculate `siteSlug`? In any case, I agree we can be flexible with where we redirect.